pages.de/common/ssh-keygen.md

```diff
@@ -0,0 +1,35 @@
+# ssh-keygen
+
+> Generiert ssh Schlüssel für Authentifizierung, Passwort-lose Logins und mehr.
+
+- Interaktives Erstellen eines SSH Schlüssel-Paars:
+
+`ssh-keygen`
+
+- Erstellen eines Schlüssel-Paars unter einem bestimmten Dateinamen:
+
+`ssh-keygen -f ~/.ssh/{{Dateiname}}`
+
+- Generieren eines ed25519 Schlüssel-Paars mit 100 Schlüssel Ableitungs-Iterationen:
+
+`ssh-keygen -t ed25519 -a 100`
+
+- Generieren eines 4096 Bit langen RSA Schlüssel-Paars mit der Email im Kommentarfeld:
+
+`ssh-keygen -t rsa -b 4096 -C "{{Email}}"`
+
+- Abrufen des Schlüssel Fingerabdrucks von einem Server (hilfreich um die Authentizität eines Servers beim ersten Verbinden zu überprüfen):
+
+`ssh-keygen -l -F {{Externer-Server}}`
```

Comment:
We use `snake_case` for variable names. Can we use this format in all pages?
```suggestion
`ssh-keygen -l -F {{Externer_Server}}`
```

Reply:
Of course 👍